src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,64 @@
+package ui;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Ui {
+    private final int INDENT_LENGTH = 4;
+    private final int LINE_LENGTH = 80;
+    private final Scanner sc = new Scanner(System.in);;
+
+    public String readLine() {
+        String userInput = sc.nextLine();
+        return userInput;
+    };
+
+    private void closeScanner() {
+        sc.close();
+    };
+
+    // Helper Function that indents text and wraps with horizontal lines
```

Comment:
Should this be a JavaDoc comment instead? E.g. something with the following form:

```suggestion
/**
 * Helper function that indents text and wraps with horizontal lines.
 * /
```

Reply:
Thanks for the spot! Will update to be a JavaDoc.